tests/test-upgrade-and-update/02-update.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo update --instance spark --namespace $NAMESPACE -p logLevel=3  --wait
```

Comment:
do you think it makes sense to extend this test-case and add more parameters just to exercise and cover more of the operator's functionality? Thinking about enabling Spark History server or metrics in the running instance of the operator and the related resources are applied or updated accordingly.

Reply:
replaced this with enabling history server